Transfer/Sources/View/TransferTableViewFooterView.swift

```diff
@@ -18,15 +18,15 @@
 
 import UIKit
 
-final class TrasferTableViewFooterView: UITableViewHeaderFooterView {
-    static let reuseIdentifier = "TrasferTableViewFooterViewIdentifier"
+final class TransferTableViewFooterView: UITableViewHeaderFooterView {
```

Comment:
why do we need custom footerView?

Reply:
https://developer.apple.com/documentation/uikit/uitableviewdatasource/1614994-tableview

The table view uses a fixed font style for section footer titles. If you want a different font style, return a custom view...

also, it is easier to implement auto-resizing, I believe